libevmasm/Assembly.cpp

```diff
@@ -242,13 +246,101 @@ string Assembly::toStringInHex(u256 _value)
 	hexStr << std::uppercase << hex << _value;
 	return hexStr.str();
 }
+vector<Json::Value> Assembly::assemblyItemAsJSON(AssemblyItem const& _item) const
+{
+	vector<Json::Value> result;
 
-Json::Value Assembly::assemblyJSON(map<string, unsigned> const& _sourceIndices) const
+	switch (_item.type())
+	{
+	case Operation:
+		result.emplace_back(createJsonValue(
+			instructionInfo(_item.instruction()).name,
+			_item));
```

Comment:
Wrapping does not match our style. Same for other `case`s in this `switch`.
```suggestion
		result.emplace_back(createJsonValue(
			instructionInfo(_item.instruction()).name,
			_item
		));
```